equinity-historical-data/src/test/resources/Crm5MockFile_5001604.txt

```diff
@@ -1821,7 +1821,7 @@
 "Originator_tl_neverpermanentlyremove": 0,
 "Ac_letters": 2,
 "Additional_text": "",
-"Ac_preparation_time": "00:15:00",
+"Ac_preparation_time": "1899-12-31T10:00:00",
```

Comment:
do we have a test for the other case?
case 1: input="00:15:00", output ="00:15:00"
case 2: input="1899-12-31T10:00:00", output="00:15:00"
case 3: input="30 minutes", output=" minutes"
case 4:  input="30 min", output="30 min"
case 5: input"100:59:00", output="00:59:00".. but it should be "100:59:00"

Reply:
Case 1 & case 2 are valid, the data in the DB suggests that Case3,4 & 5 are not valid.
I'll add two test cases.

Reply:
cool. Maybe just add a backlog ticket to add the possibility of a regex check for format. But not prio now

Reply:
Okay.  Tests added for case 1 & 2.